features/solr/src/main/feature/feature.xml

```diff
@@ -20,11 +20,15 @@
 
     <feature name="solr-dependencies" version="${project.version}" >
         <feature>pax-jetty</feature>
-        <bundle>mvn:org.eclipse.jetty/jetty-alpn-client/${jetty.version}</bundle>
-        <bundle>mvn:org.eclipse.jetty.http2/http2-hpack/${jetty.version}</bundle>
-        <bundle>mvn:org.eclipse.jetty.http2/http2-common/${jetty.version}</bundle>
-        <bundle>mvn:org.eclipse.jetty.http2/http2-client/${jetty.version}</bundle>
-        <bundle>mvn:org.eclipse.jetty.http2/http2-http-client-transport/${jetty.version}</bundle>
+        <bundle>mvn:org.eclipse.jetty/jetty-client/${solr.jetty.version}</bundle>
+        <bundle>mvn:org.eclipse.jetty/jetty-alpn-client/${solr.jetty.version}</bundle>
```

Comment:
Should we add the jetty-alpn-java-client here as well since that is actually needed for JDK 9+ environments or is that no longer needed with this upgrade?  Not all environments appear to need this but some certainly do if using the http2 client under the hood.

`<bundle>mvn:org.eclipse.jetty/jetty-alpn-java-client/${solr.jetty.version}</bundle>`

Reply:
Added 👍